Formula/libtorrent-rasterbar.rb

```diff
@@ -1,8 +1,8 @@
 class LibtorrentRasterbar < Formula
```

Comment:
Eh, sorry I spoke too soon. Do you have anymore insight into what's up with why Rasterbar was removed? Does this formula need to be renamed?

Reply:
> Do we need to rename/migrate this formula to drop references to Rasterbar?
>
> Can you find out any additional info about what happened?
>
 
It has always been just referred to as libtorrent from it's inception. The use of rasterbar was a decision made by (debian/ubuntu?) package maintainers to distinguish from [rtorrent's libTorrent](https://en.wikipedia.org/wiki/RTorrent) library but that library is afaik only used by rtorrent, compared to libtorrent that is a library used by many different clients. 

Arvid is the original author and his company is Rasterbar software so there was a link there in the beginning however I know that any reference to Rasterbar has been removed over the years from the website, email and documentation with all credits referring to Arvid.

I would say that removing the rasterbar reference would be an ideal outcome however confusion between `libtorrent` and `libTorrent` warrants keeping the naming structure and matches with linux package manager naming.

Reply:
@cas-- It looks like Debian still calls this `libtorrent-rasterbar` so let's match what they are doing. If other distros use some other naming scheme to distinguish this libtorrent from the other libtorrent, then we could add a symbolic link in the `Alias` directory with the other most common alternate naming scheme.